packages/app/features/DashboardLayout/Navigator.tsx

```diff
@@ -4,6 +4,8 @@ import { UserProfile } from '@walless/core';
 import { Compass } from '@walless/icons';
 import { ExtensionDocument } from '@walless/store';
 
+import RemoveLayout from '../../../../apps/web/src/screens/Dashboard/RemoveLayout';
```

Comment:
put RemoveLayout inside `@walless/app` instead - we'll not import `apps/*` stuffs from `packages/*`.

Reply:
In case removeLayout need access `db` or ambitious dependency, expose a Props like `onRemoveLayout` from this component and then write handler from the `apps/* side.

Reply:
Sir, In case I have to pass a function to `onRemoveLayout`, where should I write that function?